server/daemon/src/main.rs

```diff
@@ -325,104 +347,104 @@ async fn main() {
                                     }
                                     }
                                 }
-                                eprintln!("Signal received, shutting down");
+                                debug!("Signal received, shutting down");
```

Comment:
Should be an info/warn, because the user will want feedback to know the server hasn't hung and is actually responding to their reqest to shutdown :)

Reply:
That should be fixed now